### PR TITLE
upgrade BeetleX.FastHttpApi to 1.0.6

### PR DESCRIPTION
**Pull request by IKende**
upgrade BeetleX.FastHttpApi to 1.0.6

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@IKende This won't test on travis until the merge conflicts are fixed

**Comment by IKende** [comment]
Display can’t automatically merge,I don't have permission to merge pull requests.what should I do?

**Comment by nbrady-techempower** [comment]
Fix the conflicts on your branch.

**Comment by IKende** [comment]
thanks

File: .gitignore [hunk 1]
```diff
@@ -106,7 +106,11 @@ frameworks/CSharp/beetlex/Benchmarks/Properties/
 frameworks/CSharp/beetlex/Benchmarks/obj/
 frameworks/CSharp/beetlex/Benchmarks/.vs/
 frameworks/CSharp/beetlex/.vs/
+<<<<<<< HEAD
```
Comment (nbrady-techempower) [inline .gitignore]: You've left all the conflict messages in the code. Not sure how this skipped CI but these all have to be removed.

File: frameworks/CSharp/beetlex/Benchmarks/Program.cs [hunk 2]
```diff
@@ -22,21 +30,21 @@ public static void Main(string[] args)
...
             context.Response.Header[HeaderTypeFactory.DATE] = DateTime.Now.ToUniversalTime().ToString("r");
-            return new TextResult("Hello, World!");
+            return plaintextResult;
         }
 
         public object json(IHttpContext context)
         {
             context.Response.Header[HeaderTypeFactory.DATE] = DateTime.Now.ToUniversalTime().ToString("r");
-            return new JsonResult(new JsonMessage { message = "Hello, World!" });
+            return new JsonResult(jsonMessage);
```
Comment (nbrady-techempower) [inline frameworks/CSharp/beetlex/Benchmarks/Program.cs]: The jsonMessage has to be instantiated on every request per the test rules. Please revert this change.
